server/app/services/QuestionsDBServiceProvider.java

```diff
@@ -1,20 +1,20 @@
-package services.questions;
+package services;
 
 import dao.QuestionsDao;
 import services.models.UserQuestion;
 
 /**
  * Created by sabinapokhrel on 3/29/17.
  */
-public class QuestionService {
+public class QuestionsDBServiceProvider {
 
   /** This methos gets all the questions from the database
    *
    * @return
    */
   public String getAllStoredQuestions() {
     QuestionsDao questionsDao = new QuestionsDao();
-    String storedQuestions = questionsDao.getQuestions();
+    String storedQuestions = questionsDao.getQuestions().replace("[", "Here is a list of all saved questions:\n• ").replaceAll(",", "\n•").replace("]","");
```

Comment:
@dongxuDX are you able to create `Question` model here to make this return `ArrayList<Question>` instead of raw `String`?

> Does question has a model: Question? So it will be Question(Int id, String value). Then you can return ArrayList<Question> instead of String.